src/vs/platform/files/node/diskFileSystemProvider.ts

```diff
@@ -617,6 +621,7 @@ export class DiskFileSystemProvider extends Disposable implements
 						this.logService[msg.type](msg.message);
 					},
 					this.logService.getLevel() === LogLevel.Trace,
+					this.envArgs ? parseWatchServicePort(this.envArgs) : undefined,
```

Comment:
In my local testing `this.envArgs` is `undefined` , @bpasero any idea what I missed in the initialization ?

Reply:
Yes because you pass `undefined` in the disk file system provider we use on desktop.